### PR TITLE
Wp 774 TimePicker support Ethiopia 

### PR DESCRIPTION
**Pull request by DavidUM**
Issue
-------
Make timePicker support Ethiopia time

Fix
----
Current timePicker does not consider that meridiem has some entries more than two -AM & PM-
Fix related logic

Enyo-DCO-1.1-Signed-off-by: David Um david.um@lge.com

### CODE REVIEW

File: lib/TimePicker/TimePicker.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+'use strict';
```
Comment (aarontam) [inline lib/TimePicker/TimePicker.js]: Why was this added?

File: lib/TimePicker/TimePicker.js [hunk 2]
```diff
@@ -679,9 +756,7 @@ var TimePicker = module.exports = kind(
 	showPickerLabelsChanged: function (inOldvalue, inNewValue) {
 		this.$.hourLabel.set('showing', this.showPickerLabels);
 		this.$.minuteLabel.set('showing', this.showPickerLabels);
-		if(this.meridiemEnable){
-			this.$.meridiemLabel.set('showing', this.showPickerLabels);
-		}
+		if(this.meridiemEnable) this.$.meridiemLabel.set('showing', this.showPickerLabels);
```
Comment (aarontam) [inline lib/TimePicker/TimePicker.js]: Minor: let's add a space between the `if` and the `(`.

**Comment by aarontam** [comment]
Hi @DavidUM, can you fix the merge conflict - thanks! Then I can get to some squashing. :smile:

**Comment by DavidUM** [comment]
Hi @aarontam I fixed merge conflict and give a space for if statement too.

**Comment by aarontam** [comment]
Thanks @DavidUM! We will look to integrate this for the coming version, now that we have gotten through some of the release craziness.

**Comment by webOS101** [comment]
Closed in favor of https://github.com/enyojs/moonstone/pull/2479
